hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/LogEventView.avdl

```diff
@@ -0,0 +1,15 @@
+@namespace("org.hypertrace.viewgenerator.api")
+protocol LogEventViewProtocol {
+
+  record LogEventView {
+    long timestamp_nanos = 0;
+
+    union {null, string } tenant_id = null;
```

Comment:
Can you add the `name or summary` field that we discussed?

Reply:
How should it be populated?

Reply:
Look for the `message` key - https://github.com/opentracing/specification/blob/master/semantic_conventions.md